components/eam/src/physics/yaml/nucleate_ice_cam/f90_yaml/nucleate_ice_cam_calc_end_yml.f90

```diff
@@ -0,0 +1,40 @@
+#ifdef YAML_NUCICE
+  ! YAML file output generation code- DO NOT PORT to C++
+  if(yaml%flag_print) then ! if this column exists in lchnk
+
+     !write output header
+     call write_output_header(unit_output)
+
+     !start writing data
+     call write_output_var(unit_output, 'naai',  naai(yaml%col_print, yaml%lev_print))
+     call write_output_var(unit_output, 'naai_hom', naai_hom(yaml%col_print, yaml%lev_print))
```

Comment:
I believe we should write the outputs of :
nihf
niimm
nidep
nimey
See lines: 496 to 500

Reply:
added to the output